config/templates.js

```diff
@@ -3,26 +3,26 @@ module.exports = {
     owner: '18f',
     repo: 'federalist-uswds-template',
     branch: 'master',
-    example: 'https://federalist-proxy.app.cloud.gov/site/18f/federalist-uswds-template/',
-    title: 'U.S. Web Design Standards',
+    example: 'https://federalist-uswds-template.18f.gov',
+    title: 'U.S. Web Design Standards Template',
     description: 'A flexible Jekyll implementation of the U.S. Web Design Standards',
     thumb: "/images/uswds.thumb.png",
   },
   'team': {
     owner: '18f',
     repo: 'federalist-modern-team-template',
     branch: 'master',
-    example: 'https://pages.18f.gov/federalist-modern-team-template/',
-    title: 'Multi-page Template',
+    example: 'https://federalist-modern-team-template.18f.gov',
+    title: 'Multi Page Template',
     description: 'Simple Jekyll site to showcase the work of an organization',
     thumb: "/images/team.thumb.png",
   },
   'landing': {
     owner: '18f',
     repo: 'federalist-landing-page-template',
     branch: 'master',
-    example: 'http://federalist.18f.gov.s3-website-us-east-1.amazonaws.com/site/18F/federalist-landing-page-template/',
-    title: 'Single-page template',
+    example: 'https://federalist-landing-template.18f.gov',
+    title: 'Research page template',
```

Comment:
Nit: the other `title`s are in all uppercase instead of sentence case like this one. Not sure which you/we prefer, but consistency would be good.